addons/scratch-front-page-tools/addon.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "Scratch Front Page Tools",
+  "description": "Lets you hide and add new rows of Scratch front page.",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "rows",
+      "id": "rowData",
+      "type": "table",
```

Comment:
I don't think a table setting is the right idea for this addon unless you're considering expanding its functionality. Table settings allow the user to reorder rows, but the addon doesn't seem to support this. Additional rows can also be added in a table setting, but again, the addon doesn't seem to support the addition of new rows on the front page, so the user is likely to be confused by this perceived lack of functionality.

Reply:
I thought rows were reorder-able

Reply:
I tried to hard to Reorder rows but it didn't work.

Reply:
@WorldLanguages if you want I could add those buttons back because I think the settings page lack functionality.

Reply:
I don't understand, can't the rows be reordered?